app/build.gradle

```diff
@@ -28,12 +28,28 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    packagingOptions {
```

Comment:
These are "add drive storage" dependencies remove them from this PR.

Reply:
@jddeep ?

Reply:
Yes @Swati4star It is required for the aspose and grapecity library implementations and working.

Reply:
And the packaging options were provided because Android Gradle builds, you're not permitted to include the same file with the same path more than once in the output. In your build, there were two META-INF/DEPENDENCIES files coming from different places. This problem was in the Zip to Pdf fragment,